app/components/request_form/request_form.html.erb

```diff
@@ -14,6 +14,20 @@
         <% end %>
 
         <%= c 'field', object: @request, attr: :text do |field| %>
+          <div
```

Comment:
I feel like this is something that could also be a component (e.g. `toolbar`). If I remember it correctly you and @tillprochaska had some discussions on that and I am not sure what the result was. 
Anyhow nice accessibility features!

Reply:
Also like that idea, maybe it can be used/improved upon